modules/nf-scil/betcrop/fslbetcrop/main.nf

```diff
@@ -0,0 +1,68 @@
+
+process BETCROP_FSLBETCROP {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://scil.usherbrooke.ca/containers/scilus_1.5.0.sif':
+        'scilus/scilus:1.5.0' }"
+
+    input:
+        tuple val(meta), path(dwi), path(bval), path(bvec)
+
+    output:
+        tuple val(meta), path("*dwi_bet_cropped.nii.gz")            , emit: dwi
+        tuple val(meta), path("*dwi_bet_cropped_mask.nii.gz")       , emit: mask
+        path "versions.yml"                                         , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+    export ITK_GLOBAL_DEFAULT_NUMBER_OF_THREADS=1
+    export OMP_NUM_THREADS=1
+    export OPENBLAS_NUM_THREADS=1
+
+    scil_extract_b0.py $dwi $bval $bvec ${prefix}__b0.nii.gz --mean \
+        --b0_thr $params.b0_thr --force_b0_threshold
```

Comment:
We should avoid using $params to set optional arguments. Let's have a chat on how to pass them through $task.ext